src/app/core/no-content/no-content.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <h1>
+    Error - 404
+  </h1>
+  <p>An error has occured, to continue:</p>
+  <p>* Return to our <a routerLink="../">homepage</a>.<br/>
```

Comment:
It'd be better to use absolute path here to avoid potential error with redirecting to non existing path (for example after some changes)